Makefile

```diff
@@ -100,6 +100,9 @@ GVISOR_TAG ?= latest
 # auto-pause-hook tag to push changes to
 AUTOPAUSE_HOOK_TAG ?= 1.13
 
+# prow-test tag to push changes to
+PROW_TEST_TAG ?= 1.7
```

Comment:
lets change this to a sem ver format, how about v0.0.1 ?

Reply:
Done